glom/core.py

```diff
@@ -2089,6 +2089,14 @@ def chain_child(scope):
     return nxt_in_chain
 
 
+unbound_methods = set([type(str.__len__)]) #, type(Ref.glomit)])
+
+
+def _has_callable_glomit(obj):
+    glomit = getattr(obj, 'glomit', None)
+    return callable(glomit)  and not isinstance(obj, type)
```

Comment:
I was wondering if there would be any use for `classmethod` glomit -- I had one briefly at one point but then it turned out not to be a good idea

Reply:
Yeah, my thinking was the same +1